tests/unittest/_torch/thop/parallel/test_fp4_linear.py

```
@@ -86,7 +86,10 @@ def pad_up(x, pad_size):
 
 @pytest.mark.skipif(sys.version_info < (3, 12),
                     reason="cutlass-dsl 4.1.0 requires Python 3.12+")
-@skip_pre_blackwell
+@pytest.mark.skipif(
+    get_sm_version() != 100,
```

Ruff (0.12.2)

Undefined name `get_sm_version`

(F821)